oak-segment-tar/src/main/java/org/apache/jackrabbit/oak/segment/file/FileStore.java

```diff
@@ -237,6 +237,28 @@ private static GarbageCollectionStrategy newGarbageCollectionStrategy() {
         log.debug("TAR files: {}", tarFiles);
     }
 
+    private void onLockStatusChanged(SegmentNodeStorePersistence.LockStatus lockStatus) {
+        switch (lockStatus) {
+            case LOST:
+                try {
+                    if (tarFiles != null) {
+                        tarFiles.close();
```

Comment:
This will write the graph and references to the tar file (`archive.close()` in `TarWriter`), shouldn't we avoid that? Also it will try to flush the write queue (`q.flush()` in `AbstractRemoteSegmentArchiveWriter`), so if it's not empty, it will wait there forever, right?

Reply:
Right. Thanks for spotting this, @ahanikel !
I suppose we should add an `unsafeShutdown()` method to `TarFiles`, `TarWriter` and `AbstractRemoteSegmentArchiveWriter`, so that we can shut them down without attempting any finalisation writes to the remote storage, which would anyway fail or corrupt the data.
What do you think?